### PR TITLE
Clarification of degrees/radians in angle methods

### PR DESCRIPTION
**Pull request by cbscribe**
Added and/or clarified angle-based methods to indicate use of radians or degrees.

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -10865,12 +10865,14 @@
 			<return type="float">
 			</return>
 			<description>
+                Return the rotation (in radians)
```
Comment (akien-mga) [inline doc/base/classes.xml]: You seem to have an indentation issue, it should be using tabs, not spaces.

**Comment by cbscribe** [comment]
Should be cleared up now.  Fixed the tabs (and the badly configured Atom git plugin).

**Comment by akien-mga** [comment]
Would you be able to squash those commits together? Check `git rebase -i HEAD~4`.

**Comment by akien-mga** [comment]
Once you've rebased, you need to force push with `git push --force`, otherwise it will ask you to merge again and again and you'll end up with an endless number of commits (like 7 right now).

**Comment by cbscribe** [comment]
Alright tried that.  Sorry about the headache.  Should I just nuke the whole thing? I don't want to make more work for you.

**Comment by akien-mga** [comment]
In the end I cherry-picked the relevant changes manually in d914984. Sorry about that, git can be difficult to tame :)

**Pull request closed without merging**